### PR TITLE
Bg fix environment setup db 167848124

### PR DESCRIPTION
**Pull request by WinnersProx**
#### What does this PR do?
- environment setup
#### Description of Task to be completed?
- add Postgres file to travis.yml file
- add environment files in `.env` file
#### How should this be manually tested?
- By just cloning the app and reading the documentation [here](https://way-farer-app-rest.herokuapp/api/v2/api-docs)
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/167848124

### CODE REVIEW

File: server/v2/middlewares/auth_middleware.js [hunk 1]
```diff
@@ -58,7 +58,6 @@ export default  {
   },
   exists : async (req, res, next) => {
     const user = await User.userExists(req.body.email);
```
Comment (oesukam) [inline server/v2/middlewares/auth_middleware.js:60]: @WinnersProx Destruct the email from body

File: server/v2/config/db.config.js [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
-  host: 'localhost',
-  database: 'way_farer_db',
-  password: 'winners1',
-  port: 5432
-});
+import dotenv from 'dotenv';
+dotenv.config();
+let pool;
+if(process.env.NODE_ENV === 'production'){
+	const connectionString = process.env.DATABASE_URL;
```
Comment (oesukam) [inline server/v2/config/db.config.js:6]: @WinnersProx Destruct the DATABASE_URL from `process.env`

**Review by oesukam — CHANGES_REQUESTED** [review]
@WinnersProx Avoid using `ES5` in the code base as `ES6` is the minimum version to use

**Comment by WinnersProx** [comment]
already fixed!

**Pull request closed without merging**